source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -84,6 +84,9 @@ To launch the OZWCP web application:
   Don't check the USB box regardless of using a USB based device.
 </p>
 
+### {% linkable_title Using the GPIOs %}
```

Comment:
I've never had changes added to a pull request.  is there something I need to do to further this?